cmd/machine-config-operator/bootstrap.go

```diff
@@ -111,6 +114,7 @@ func runBootstrapCmd(cmd *cobra.Command, args []string) {
 			MdnsPublisher:       bootstrapOpts.mdnsPublisherImage,
 			Haproxy:             bootstrapOpts.haproxyImage,
 			BaremetalRuntimeCfg: bootstrapOpts.baremetalRuntimeCfgImage,
+			ReleaseImage:        bootstrapOpts.releaseImage,
```

Comment:
this isn't an "image" as we mean about the others here (I get the confusion tho :( ) - we'll need to pass this down to RenderBootstrap directly - so, I think this PR is enough by just having the dummy flag added and we can proceed once we have a meaningful value there

Reply:
let's get this in w/o changing this also :) this is ok for no 👍

Reply:
Fixed as well :)